ffmpeg/plan.sh

```diff
@@ -34,15 +46,25 @@ do_build() {
     --disable-static \
     --enable-avisynth \
     --enable-avresample \
+    --disable-stripping \
+    --enable-fontconfig \
     --enable-gmp \
+    --enable-gnutls \
     --enable-gpl \
+    --enable-libdrm \
+    --enable-libfreetype \
     --enable-libopenjpeg \
     --enable-libwebp \
+    --enable-libxml2 \
     --enable-shared \
     --enable-version3
   make -j"$(nproc)"
 }
 
 do_check() {
+  local ffmpeg_path="$HAB_CACHE_SRC_PATH/${pkg_dirname}"
```

Comment:
Would you mind adding the following comment? 
```
# ffmpeg links against itself and due to the nature of the Habitat build process, expects those libraries
# to be present at /hab/pkgs/<pkg_ident> .  Since do_check is called before do_install, those libraries are 
# not yet present, causing the tests to fail.  We use LD_LIBRARY_PATH here to work around this chicken/egg
# scenario, to provide the build paths of the libraries _only_ in the context of the do_check.
```

Typically we don't use LD_LIBRARY_PATH in Habitat builds except in very specific scenarios, like this one. The above comment will remind future maintainers why its use is required here.